plantshop.py

```diff
@@ -18,7 +28,7 @@ def wither(self, how_many_leaves_fall):
 
 rose.wither(5)
 rose.grow(15)
+cactus.propagate(3)
 
```

Comment:
Can we add some more test cases besides 1.

Reply:
will do that in the next round